uber/models.py

```diff
@@ -1324,6 +1348,22 @@ def past_years_json(self):
         return json.loads(self.past_years or '[]')
 
 
+class WatchList(MagModel):
```

Comment:
I would have expected the foreign key relationship to be the other way around.  Like, if "John Smith" is banned, then I'd expect to have a row in this table like

```
first_names => "John,Jonathan"
last_name => "Smith"
email => ... (etc)
```

And then anytime someone with that info registers, we'd link the two entries.  I can imagine multiple ``Attendee`` entries linking to the same ``Watchlist`` (since people sometimes register multiple times) but I don't see a use case for it being the other way around.  Therefore, my intuition is that ``Attendee`` should foreign key into ``Watchlist`` instead of vice versa.

Does that make sense?  Or am I not thinking it all the way through?